### PR TITLE
Add XunLei.xunlei11 version 11.2.4.1750

### PR DESCRIPTION
**Pull request by Voya9e**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20469)

### CODE REVIEW

File: manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Platform: 
+ - "Windows.Desktop"
+MinimumOSVersion: ""
+InstallerType: "exe"
+InstallModes: 
+ - "silent"
+Installers: 
+- Architecture: x86
+  InstallerUrl: https://down.sandai.net/thunder11/XunLeiSetup11.2.4.1750.exe
+  InstallerSha256: C2D826F9410315AD4EB12480932947C534810DD8224B55DF5C21F85C3905C7E3
```
Comment (ItzLevvie) [inline manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.installer.yaml:12]:
```suggestion
  InstallerSha256: C2D826F9410315AD4EB12480932947C534810DD8224B55DF5C21F85C3905C7E3
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+PackageIdentifier: "XunLei.xunlei11"
```
Comment (ItzLevvie) [inline manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.installer.yaml:1]:
```suggestion
PackageIdentifier: "XunLei.xunlei"
```
Oops, I forgot this.

File: manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: "XunLei.xunlei11"
```
Comment (ItzLevvie) [inline manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.locale.zh-CN.yaml:1]:
```suggestion
PackageIdentifier: "XunLei.xunlei"
```
Oops, I forgot this.

File: manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: "XunLei.xunlei11"
```
Comment (ItzLevvie) [inline manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.yaml:1]:
```suggestion
PackageIdentifier: "XunLei.xunlei"
```
Oops, I forgot this.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please make sure to update the version folder as well; using GitHub Desktop.

**Comment by Meng5in** [comment]
> Please make sure to update the version folder as well; using GitHub Desktop.

Well, it's too late now, I'll do it tomorrow.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**